src/css/main.less

```diff
@@ -1478,3 +1472,7 @@ h3.small-caps {
     padding: 0;
   }
 }
+
+.task-details dt {
+  text-align: left;
```

Comment:
Maybe this should be applied in a generic directly on the dt, so it affects all dt's ?
![taskdeatils](https://cloud.githubusercontent.com/assets/859154/10516543/1156cb78-7359-11e5-9b00-c8a266881863.png)
Also the AppVersionComponent needs this.